src/components/Header/index.tsx

```diff
@@ -24,35 +27,45 @@ export default function Header() {
     Prism.highlightAll()
   }, [])
 
-  const handleToggleMobileMenu = () => {
+  const handleOpenMobileMenu = () => {
```

Comment:
Why it's renamed? The logic inside seems like a toggle for me

Reply:
This happened on a master merge, though I'm not sure why it was marked as a change in that case.

Reply:
Reverted.